Gems/Atom/RPI/Code/Include/Atom/RPI.Public/AuxGeom/AuxGeomDraw.h

```diff
@@ -256,6 +257,17 @@ namespace AZ
             //! @param faceCull      Which (if any) facing triangles should be culled
             //! @param viewProjOverrideIndex Which view projection override entry to use, -1 if unused
             virtual void DrawObb(const AZ::Obb& obb, const AZ::Matrix3x4& transform, const AZ::Color& color, DrawStyle style = DrawStyle::Shaded, DepthTest depthTest = DepthTest::On, DepthWrite depthWrite = DepthWrite::On, FaceCullMode faceCull = FaceCullMode::Back, int32_t viewProjOverrideIndex = -1) = 0;
+
+            //! Draw a frustum.
+            //! @param frustum       The frustum
+            //! @param color         The color to draw the frustum
+            //! @param drawNormals   If true, frustum plane normals will be drawn as lines
+            //! @param style         The draw style (point, wireframe, solid, shaded etc)
```

Comment:
I actually had periods to start with but none of the other functions in this class did, so I opted to follow the local style instead of being the odd one out.

Reply:
Ah okay understood, makes sense thanks 👍